keyboards/40percentclub/25/keymaps/macro/keymap.c

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2018 REPLACE_WITH_YOUR_NAME
```

Comment:
```suggestion
/* Copyright 2018 zvecr
```

Reply:
On other boards, i have left it as `Copyright 2018`, is there a standard for community supported boards? And as other 40percent.club boards do not have a copyright notice, if i were to add one, i wouldnt want to put my own name against someone elses work?